server/routes/slack/slack_events_helpers.py

```
@@ -59,6 +59,48 @@ def verify_slack_signature(request_body: bytes, timestamp: str, signature: str)
+                        WHERE provider = 'slack' 
+                        AND subscription_id = %s
+                        AND is_active = TRUE
+                        """,
+                        (team_id,)
+                    )
+                    results = cursor.fetchall()
+                    if results:
+                        return results
+        except Exception as e:
+            logger.debug(f"Error querying slack tokens for org {org_id}: {e}")
+            continue
```

Ruff (0.15.15)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)